todo

. Solve this error : [ValidationError(['Account with this Phone number already exists.'])]





-----------------------------------
done
. setup a django project
. change mysite to src
. create home app
. set phone number as foreign key with account table